base/compiler/ssair/inlining.jl

```diff
@@ -533,6 +550,12 @@ function batch_inline!(todo::Vector{Pair{Int, Any}}, ir::IRCode, linetable::Vect
         for ((old_idx, idx), stmt) in compact
             if old_idx == inline_idx
                 argexprs = copy(stmt.args)
+                if isa(item, InliningTodo) && item.mi === nothing
```

Comment:
this again makes me this we should use a different marker than `nothing` to indicate that this is a opaque closure.

Reply:
There are MethodInstances for opaque closures at this stage, inlining just doesn't support that yet.